I.write-tests/06-largest-number/largest-number.test.js

```diff
@@ -1,5 +1,18 @@
+test("Largest number", function() {
+  //Arrange
+  var largestNumber = require("./largest-number");
+  var numbers = [3, 21, 88, 4, 36];
+  //Act
+
+  var result = largestNumber(numbers);
+
+  //Assert
+  expect(result).toEqual(88);
+  expect(numbers).toEqual([3, 21, 88, 4, 36]);
```

Comment:
Does this test pass? If not, why not?

Reply:
Also, how can you make sure the test passes for all cases? Can you add more test cases?